--- max78000.svd.original	2024-01-18 04:56:16.082733600 -0800
+++ max78000.svd.patched	2024-01-18 05:43:10.469660500 -0800
@@ -2950,125 +2950,165 @@
         </enumeratedValue>
        </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>WDT0</name>
        <description>Watchdog Timer 0 Reset.</description>
        <bitOffset>1</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>GPIO0</name>
        <description>GPIO0 Reset. Setting this bit to 1 resets GPIO0 pins to their default states.</description>
        <bitOffset>2</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>GPIO1</name>
        <description>GPIO1 Reset. Setting this bit to 1 resets GPIO1 pins to their default states.</description>
        <bitOffset>3</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>TMR0</name>
        <description>Timer 0 Reset. Setting this bit to 1 resets Timer 0 blocks.</description>
        <bitOffset>5</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>TMR1</name>
        <description>Timer 1 Reset. Setting this bit to 1 resets Timer 1 blocks.</description>
        <bitOffset>6</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>TMR2</name>
        <description>Timer 2 Reset. Setting this bit to 1 resets Timer 2 blocks.</description>
        <bitOffset>7</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>TMR3</name>
        <description>Timer 3 Reset. Setting this bit to 1 resets Timer 3 blocks.</description>
        <bitOffset>8</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>UART0</name>
        <description>UART 0 Reset. Setting this bit to 1 resets all UART 0 blocks.</description>
        <bitOffset>11</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>UART1</name>
        <description>UART 1 Reset. Setting this bit to 1 resets all UART 1 blocks.</description>
        <bitOffset>12</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>SPI1</name>
        <description>SPI 1 Reset. Setting this bit to 1 resets all SPI 1 blocks.</description>
        <bitOffset>13</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>I2C0</name>
        <description>I2C 0 Reset.</description>
        <bitOffset>16</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>RTC</name>
        <description>Real Time Clock Reset.</description>
        <bitOffset>17</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>SMPHR</name>
        <description>Semaphore Reset.</description>
        <bitOffset>22</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>TRNG</name>
        <description>TRNG Reset. This reset is only available during the manufacture testing phase.</description>
        <bitOffset>24</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>CNN</name>
        <description>CNN Reset.</description>
        <bitOffset>25</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>ADC</name>
        <description>ADC Reset.</description>
        <bitOffset>26</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>UART2</name>
        <description>UART2 Reset. Setting this bit to 1 resets all UART 2 blocks.</description>
        <bitOffset>28</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>SOFT</name>
        <description>Soft Reset. Setting this bit to 1 resets everything except the CPU and the watchdog timer.</description>
        <bitOffset>29</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>PERIPH</name>
        <description>Peripheral Reset. Setting this bit to 1 resets all peripherals. The CPU core, the watchdog timer, and all GPIO pins are unaffected by this reset.</description>
        <bitOffset>30</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="DMA">
+      <field>
        <name>SYS</name>
        <description>System Reset. Setting this bit to 1 resets the CPU core and all peripherals, including the watchdog timer.</description>
        <bitOffset>31</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -3681,8 +3721,8 @@
        <bitOffset>0</bitOffset>
        <bitWidth>1</bitWidth>
        <enumeratedValues>
-        <name>reset_read</name>
-        <usage>read</usage>
+        <name>reset</name>
+        <usage>read-write</usage>
         <enumeratedValue>
          <name>reset_done</name>
          <description>Reset complete.</description>
@@ -3695,71 +3735,95 @@
         </enumeratedValue>
        </enumeratedValues>
       </field>
-      <field derivedFrom="I2C1">
+      <field>
        <name>PT</name>
        <description>PT Reset.</description>
        <bitOffset>1</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="I2C1">
+      <field>
        <name>OWM</name>
        <description>OWM Reset.</description>
        <bitOffset>7</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="I2C1">
+      <field>
        <name>CRC</name>
        <description>CRC Reset.</description>
        <bitOffset>9</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="I2C1">
+      <field>
        <name>AES</name>
        <description>AES Reset.</description>
        <bitOffset>10</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="I2C1">
+      <field>
        <name>SPI0</name>
        <description>SPI 0 Reset.</description>
        <bitOffset>11</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="I2C1">
+      <field>
        <name>SMPHR</name>
        <description>SMPHR Reset.</description>
        <bitOffset>16</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="I2C1">
+      <field>
        <name>I2S</name>
        <description>I2S Reset.</description>
        <bitOffset>19</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="I2C1">
+      <field>
        <name>I2C2</name>
        <description>I2C2 Reset.</description>
        <bitOffset>20</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="I2C1">
+      <field>
        <name>DVS</name>
        <description>DVS Reset.</description>
        <bitOffset>24</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="I2C1">
+      <field>
        <name>SIMO</name>
        <description>SIMO Reset.</description>
        <bitOffset>25</bitOffset>
        <bitWidth>1</bitWidth>
+       <enumeratedValues derivedFrom="reset">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="I2C1">
+      <field>
        <name>CPU1</name>
        <description>CPU1 Reset.</description>
        <bitOffset>31</bitOffset>
        <bitWidth>1</bitWidth>
+       <access>read-only</access>
+       <enumeratedValues derivedFrom="reset">
+        <usage>read</usage>
+       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4145,18 +4209,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>ALTERNATE</name>
-         <description>Alternate function enabled.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>GPIO</name>
-         <description>GPIO function is enabled.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4196,18 +4248,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>dis</name>
-         <description>GPIO Output Disable</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>en</name>
-         <description>GPIO Output Enable</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4247,18 +4287,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>low</name>
-         <description>Drive Logic 0 (low) on GPIO output.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>high</name>
-         <description>Drive logic 1 (high) on GPIO output.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4273,18 +4301,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>no</name>
-         <description>No Effect.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>set</name>
-         <description>Set GPIO_OUT bit in this position to '1'</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4326,18 +4342,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>level</name>
-         <description>Interrupts for this pin are level triggered.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>edge</name>
-         <description>Interrupts for this pin are edge triggered.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4351,18 +4355,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>falling</name>
-         <description>Interrupts are latched on a falling edge or low level condition for this pin.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>rising</name>
-         <description>Interrupts are latched on a rising edge or high condition for this pin.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4370,6 +4362,14 @@
      <name>INEN</name>
      <description>GPIO Input Enable</description>
      <addressOffset>0x30</addressOffset>
+     <fields>
+      <field>
+       <name>GPIO_INEN</name>
+       <description>Mask of all of the pins on the port.</description>
+       <bitOffset>0</bitOffset>
+       <bitWidth>32</bitWidth>
+      </field>
+     </fields>
     </register>
     <register>
      <name>INTEN</name>
@@ -4381,18 +4381,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>dis</name>
-         <description>Interrupts are disabled for this GPIO pin.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>en</name>
-         <description>Interrupts are enabled for this GPIO pin.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4406,18 +4394,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>no</name>
-         <description>No effect.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>set</name>
-         <description>Set GPIO_INT_EN bit in this position to '1'</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4431,18 +4407,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>no</name>
-         <description>No Effect.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>clear</name>
-         <description>Clear GPIO_INT_EN bit in this position to '0'</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4457,18 +4421,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>no</name>
-         <description>No Interrupt is pending on this GPIO pin.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>pending</name>
-         <description>An Interrupt is pending on this GPIO pin.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4495,18 +4447,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>dis</name>
-         <description>PMU wakeup for this GPIO is disabled.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>en</name>
-         <description>PMU wakeup for this GPIO is enabled.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4546,18 +4486,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>no</name>
-         <description>No Effect.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>en</name>
-         <description>Dual Edge mode is enabled. If edge-triggered interrupts are enabled on this GPIO pin, then both rising and falling edges will trigger interrupts regardless of the GPIO_INT_POL setting.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4568,26 +4496,9 @@
      <fields>
       <field>
        <name>GPIO_PADCTRL0</name>
-       <description>The two bits in GPIO_PAD_CFG1 and GPIO_PAD_CFG2 for each GPIO pin work together to determine the pad mode when the GPIO is set to input mode.</description>
+       <description>The two bits in GPIO_PADCTRL0 and GPIO_PADCTRL1 for each GPIO pin work together to determine the pad mode when the GPIO is set to input mode.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>impedance</name>
-         <description>High Impedance.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>pu</name>
-         <description>Weak pull-up mode.</description>
-         <value>1</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>pd</name>
-         <description>weak pull-down mode.</description>
-         <value>2</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4598,26 +4509,9 @@
      <fields>
       <field>
        <name>GPIO_PADCTRL1</name>
-       <description>The two bits in GPIO_PAD_CFG1 and GPIO_PAD_CFG2 for each GPIO pin work together to determine the pad mode when the GPIO is set to input mode.</description>
+       <description>The two bits in GPIO_PADCTRL0 and GPIO_PADCTRL1 for each GPIO pin work together to determine the pad mode when the GPIO is set to input mode.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>impedance</name>
-         <description>High Impedance.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>pu</name>
-         <description>Weak pull-up mode.</description>
-         <value>1</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>pd</name>
-         <description>weak pull-down mode.</description>
-         <value>2</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4631,18 +4525,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>primary</name>
-         <description>Primary function selected.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>secondary</name>
-         <description>Secondary function selected.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4682,18 +4564,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>primary</name>
-         <description>Primary function selected.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>secondary</name>
-         <description>Secondary function selected.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4746,18 +4616,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>FAST</name>
-         <description>Fast Slew Rate selected.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>SLOW</name>
-         <description>Slow Slew Rate selected.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -4771,18 +4629,6 @@
        <description>Mask of all of the pins on the port.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>32</bitWidth>
-       <enumeratedValues>
-        <enumeratedValue>
-         <name>ld</name>
-         <description>GPIO port pin is in low-drive mode.</description>
-         <value>0</value>
-        </enumeratedValue>
-        <enumeratedValue>
-         <name>hd</name>
-         <description>GPIO port pin is in high-drive mode.</description>
-         <value>1</value>
-        </enumeratedValue>
-       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -6833,9 +6679,10 @@
        <description>Low Power GPIO 2 Reset.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>1</bitWidth>
+       <access>write-only</access>
        <enumeratedValues>
-        <name>reset</name>
-        <usage>read-write</usage>
+        <name>reset_write</name>
+        <usage>write</usage>
         <enumeratedValue>
          <name>reset_done</name>
          <description>Reset complete.</description>
@@ -6848,35 +6695,50 @@
         </enumeratedValue>
        </enumeratedValues>
       </field>
-      <field derivedFrom="GPIO2">
+      <field>
        <name>WDT1</name>
        <description>Low Power Watchdog Timer 1 Reset.</description>
        <bitOffset>1</bitOffset>
        <bitWidth>1</bitWidth>
+       <access>write-only</access>
+       <enumeratedValues derivedFrom="reset_write">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="GPIO2">
+      <field>
        <name>TMR4</name>
        <description>Low Power Timer 4 Reset.</description>
        <bitOffset>2</bitOffset>
        <bitWidth>1</bitWidth>
+       <access>write-only</access>
+       <enumeratedValues derivedFrom="reset_write">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="GPIO2">
+      <field>
        <name>TMR5</name>
        <description>Low Power Timer 5 Reset.</description>
        <bitOffset>3</bitOffset>
        <bitWidth>1</bitWidth>
+       <access>write-only</access>
+       <enumeratedValues derivedFrom="reset_write">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="GPIO2">
+      <field>
        <name>UART3</name>
        <description>Low Power UART 3 Reset.</description>
        <bitOffset>4</bitOffset>
        <bitWidth>1</bitWidth>
+       <access>write-only</access>
+       <enumeratedValues derivedFrom="reset_write">
+       </enumeratedValues>
       </field>
-      <field derivedFrom="GPIO2">
+      <field>
        <name>LPCOMP</name>
        <description>Low Power Comparator Reset.</description>
        <bitOffset>6</bitOffset>
        <bitWidth>1</bitWidth>
+       <access>write-only</access>
+       <enumeratedValues derivedFrom="reset_write">
+       </enumeratedValues>
       </field>
      </fields>
     </register>
@@ -12577,6 +12439,26 @@
          <description>The second value to be written to reset the WDT.</description>
          <value>0x0000005A</value>
         </enumeratedValue>
+        <enumeratedValue>
+         <name>seq2</name>
+         <description>WDT Enable: first rst value.</description>
+         <value>0x000000FE</value>
+        </enumeratedValue>
+        <enumeratedValue>
+         <name>seq3</name>
+         <description>WDT Enable: second rst value.</description>
+         <value>0x000000ED</value>
+        </enumeratedValue>
+        <enumeratedValue>
+         <name>seq4</name>
+         <description>WDT Disable: first rst value.</description>
+         <value>0x000000DE</value>
+        </enumeratedValue>
+        <enumeratedValue>
+         <name>seq5</name>
+         <description>WDT Disable: second rst value.</description>
+         <value>0x000000AD</value>
+        </enumeratedValue>
        </enumeratedValues>
       </field>
      </fields>
